### DISCUSSION TITLE
Potential issue

### DISCUSSION THREAD

**Discussion by tungthncc** [fishaudio/Bert-VITS2#249] [General]
```
Module 'attentions' has no 'FFT' member; maybe 'FFN'?
at models.py, line 107
```

**Comment by Stardust-minus** [comment]
No.We didn't implement the share parameter.That's the legacy problem code

**Reply by tungthncc** [reply]
I see. So, can it raise any issues in the future? Do you plan to do anything about that?

**Reply by Stardust-minus** [reply]
> I see. So, can it raise any issues in the future? Do you plan to do anything about that?

Maybe we can delete it in the future.For now,if you don't modify the code,it won't cause any problem.

**Reply by tungthncc** [reply]
Yeah. Thank you!
